src/tic_tac_toe/board.clj

```diff
@@ -2,10 +2,30 @@
 
 (def space " ")
 
-(defn init []
-  vector (replicate 9 space))
+(defn get-size [width]
+	(* width width))
+
+(defn get-new-board [width]
+  vector (replicate (get-size width) space))
```

Comment:
Just checked. if I put paren around the vector function and run the tests i get the following:
```
Failures:

  1) create empty board should create a board with size of 9 if width is 3
     Expected: [" " " " " " " " " " " " " " " " " "]
          got: [(" " " " " " " " " " " " " " " " " ")] (using =)
     /Users/richardcheng/Documents/clojure-tic-tac-toe/spec/tic_tac_toe/board_spec.clj:20

  2) create empty board should create a board with size of 16 if width is 4
     Expected: [" " " " " " " " " " " " " " " " " " " " " " " " " " " " " " " "]
          got: [(" " " " " " " " " " " " " " " " " " " " " " " " " " " " " " " ")] (using =)
     /Users/richardcheng/Documents/clojure-tic-tac-toe/spec/tic_tac_toe/board_spec.clj:22

Finished in 0.00970 seconds
13 examples, 2 failures
```